Move stuff from README to docs (#436)
Uninett/zino: README.md

[![build badge](https://img.shields.io/github/actions/workflow/status/Uninett/zino/tests.yml?branch=master)](https://github.com/Uninett/zino/actions)
[![codecov badge](https://codecov.io/gh/Uninett/zino/branch/master/graph/badge.svg)](https://codecov.io/gh/Uninett/zino)
[![Code style: Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
[![rtfd badge](https://app.readthedocs.org/projects/zino/badge/?version=latest&style=flat)](https://zino.readthedocs.io/en/latest/)
[![Python 3.9](https://img.shields.io/badge/python-3.9-blue.svg)](https://www.python.org/downloads/release/python-3922/)
[![Python 3.10](https://img.shields.io/badge/python-3.10-blue.svg)](https://www.python.org/downloads/release/python-31018/)

> [![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)](https://www.python.org/downloads/release/python-31113/)

Reviewer: Forgot 3.12 here

Reviewer: No it wasn't. Python 3.12 has been there since the 2.0.0 release. It just wasn't mentioned explicitly in the README :grin:

## Installing Zino

> Yoou need a running Python of a supported version, and these days a virtualenv

Reviewer: ```suggestion
You need a supported Pythonv ersion, and these days a virtualenv
```
